tests/quantile.rs

```diff
@@ -5,10 +5,21 @@ extern crate ndarray_stats;
 use ndarray::prelude::*;
 use ndarray_stats::{
     interpolate::{Higher, Linear, Lower, Midpoint, Nearest},
-    QuantileExt,
-    Quantile1dExt,
+    Quantile1dExt, QuantileExt,
 };
 
+#[test]
+fn test_argmin() {
+    let a = array![[0, 5, 3], [2, 0, 6]];
+    assert_eq!(a.argmin(), Some((0, 0)));
+
+    let a = array![[1., 5., 3.], [2., 0., 6.]];
+    assert_eq!(a.argmin(), Some((1, 1)));
+
+    let a = array![[1., 5., 3.], [2., ::std::f64::NAN, 6.]];
+    assert_eq!(a.argmin(), None);
+}
```

Comment:
We should add a test case, both for `argmin` and `argmax`, taking an empty array as input.